Tests/ClientTests/Frontend/Home/Pocket/PocketStoryProviderTests.swift

```diff
@@ -11,236 +11,27 @@ class PocketStoryProviderTests: XCTestCase, FeatureFlaggable {
 
     override func setUp() {
         super.setUp()
-        LegacyFeatureFlagsManager.shared.initializeDeveloperFeatures(with: MockProfile())
     }
 
     override func tearDown() {
         super.tearDown()
         subject = nil
     }
 
-    func testIfSponsoredAreDisabled_FetchingStories_ReturnsTheNonSponsoredList() async {
-        featureFlags.set(feature: .sponsoredPocket, to: false)
+    func testIfSponsoredAreDisabled_FetchingStories_ReturnsList() async {
```

Comment:
The name speak about sponsored tiles, wondering if we should rename to something else
```suggestion
    func testIfSponsoredAreDisabled_FetchingStories_ReturnsList() async {
```